src/components/nowplayingbar/nowplayingbar.js

```diff
@@ -397,11 +387,9 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
             showMuteButton = false;
         }
 
-        if (isMuted) {
-            muteButton.querySelector('i').innerHTML = '&#xE04F;';
-        } else {
-            muteButton.querySelector('i').innerHTML = '&#xE050;';
-        }
+        let icons = ['volume_off', 'volume_up'];
+        if (isMuted) icons = icons.reverse();
+        muteButton.querySelector('.material-icons').classList.replace(icons[0], icons[1]);
```

Comment:
this looks like a need for a helper method probably... also what happens if `icons[0]` is not present in the `.classList`?

Reply:
> also what happens if `icons[0]` is not present in the `.classList`?

It won't work - `replace` requires a "target".

Another approach is
- remove all possible classes
- add current state class

```suggestion
        const icon = muteButton.querySelector('.material-icons');
        icon.classList.remove('volume_off', 'volume_up');
        icon.classList.add(isMuted ? 'volume_off' : 'volume_up');
```

Reply:
I like this more, let's use this pattern.